README.md

```diff
@@ -42,42 +42,63 @@ for func in d:
 It is recommended to first refer the [NetworkX's INSTALL.rst](https://github.com/networkx/networkx/blob/main/INSTALL.rst).
 nx-parallel requires Python >=3.10. Right now, the only dependencies of nx-parallel are networkx and joblib.
 
-### Install the released version
+### Installing nx-parallel using `pip`
 
 You can install the stable version of nx-parallel using pip:
 
-```sh
+```.sh
```

Comment:
I've never seen this before... what's the `.` for?

Reply:
It's nothing. 
I think adding a `.` was making my `.md` file error free in vscode. So, I added it without thinking much about it. But I think it was probably because the code block was not getting associated with python.

I have removed it. Thanks :)
And Sorry, I'll be more careful in the future.

Reply:
No worries, I thought it might be some new syntax highlighting feature :upside_down_face: